### PR TITLE
Sdk js to v1.5.8 and php to 1.4.1

### PR DESCRIPTION
**Pull request by vutreras**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.0.2] - 2018-10-29
+### Changed
+- Se actualiza sdk js a versión 1.5.8
```
Comment (leosoto) [inline CHANGELOG.md]: Someone reading this changelog won't really know what changed nor why should he upgrade. Please phrase the changelog in a way that the user of plugins can understand (and note that plugins are used by people who are not developers)

File: README.md [hunk 2]
```diff
@@ -78,3 +82,9 @@ rm -rf app/code/Transbank/Onepay*
 ```
 
 3. Seguir el proceso de instalación descrito anteriormente.
+
+## Desarrollo
+
+Para apoyar el levantamiento rápido de un ambiente de desarrollo, hemos creado la especificación de contenedores a través de Docker Compose.
+
+Para usarlo seguir el siguiente [README Magento 2](./docker-magento3)
```
Comment (leosoto) [inline README.md]: I think that `3` should be a `2`.

File: CHANGELOG.md (hunk 1 above)
Comment (leosoto) [inline CHANGELOG.md]: Someone reading this changelog won't really know what changed nor why should he upgrade. Please phrase the changelog in a way that the user of plugins can understand (and note that plugins are used by people who are not developers)

File: README.md (hunk 2 above)
Comment (leosoto) [inline README.md]: I think that 3 should be a 2.

File: docker-magento2/.docker/bin/fix-permission [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+chmod -R 777 /var/www/
+# find /var/www/html/ -type f -exec chmod 644 {} \;      # 644 permission for files
```
Comment (leosoto) [inline docker-magento2/.docker/bin/fix-permission]: Don't commit commented out code. (unless there is a very good reason, but no good reason is explained here)

File: docker-magento2/docs/INSTALLATION.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
+# Manual de instalación para Plugin Magento2
```
Comment (leosoto) [inline docker-magento2/docs/INSTALLATION.md:1]: If this is new/updated please remember to update it the transbank-developers-doc repository too

**Review by leosoto — CHANGES_REQUESTED** [review]

File: docker-magento2/docs/INSTALLATION.md (hunk 4 above)
Comment (vutreras) [reply docker-magento2/docs/INSTALLATION.md:1]: This doc is only for developers, is a help to start up environment of magento2

**Review by leosoto — APPROVED** [review]

**Pull request merged**